stable/redis/values.yaml

```diff
@@ -172,6 +172,8 @@ master:
   ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#taints-and-tolerations-beta-feature
   # nodeSelector: {"beta.kubernetes.io/arch": "amd64"}
   # tolerations: []
+  ## Redis Master pod/node affinity/anti-affinity
+  # affinity: {}
```

Comment:
Leave it uncommented. See how it's implemented on "slave" section

Reply:
Fixed.